src/rpc/methods/f3/types.rs

```diff
@@ -267,6 +281,34 @@ pub struct ChainExchangeConfig {
     pub max_timestamp_age: Duration,
 }
 
+#[derive(Educe)]
```

Comment:
Can you please elaborate on why we need `educe`? What is the alternative?

Reply:
`educe` is used for setting default field values. The alternative would be implementing `Default` manually.

Reply:
Unless I'm missing something, implementing it manually makes more sense:
1. Less dependencies
2. Less mental overhead (devs don't have to google `educe`, everyone knows how to implement `Default`)
3. Not that much more code, maybe even less, given that for every field you are adding two lines of annotations.

Reply:
@LesnyRumcajs defaults are no longer needed and removed as discussed [here](https://filecoinproject.slack.com/archives/C0556MSR945/p1742262567890869).

> given that for every field you are adding two lines of annotations.

One line for `educe` and one line for `serde`, we still need the `serde` one when implementing `Default` manually